pkg/cloudprovider/aws/packing/packing.go

```diff
@@ -78,6 +80,42 @@ func (p *podPacker) Pack(ctx context.Context, pods []*v1.Pod) ([]*Packing, error
 
 // TODO filter instance types based on node contraints like availability zones etc.
 func (p *podPacker) getNodeCapacities() []*nodeCapacity {
+	nodeCapacities := []*nodeCapacity{}
+	describeInstanceTypesInput := &ec2.DescribeInstanceTypesInput{
+		Filters: []*ec2.Filter{
+			{
+				Name:   aws.String("processor-info.supported-architecture"),
+				Values: []*string{aws.String("x86_64")},
```

Comment:
we do also support arm, although I can add that in a later PR

Reply:
oh cool, I didn't see that. I can add it in here!